### ISSUE TITLE
What's wrong with the anchor point of photo_view?

### ISSUE THREAD

**Issue by ProgrammingLife** [renancaraujo/photo_view#174]
I use pretty simple code:
```
return Scaffold(
      appBar: AppBar(
        title: Text('Simple Photo View'),
      ),
      body: PhotoView(
        imageProvider: AssetImage("assets/images/image.png"),
        //minScale: PhotoViewComputedScale.contained * 0.8,
        //maxScale: PhotoViewComputedScale.covered * 2,
        enableRotation: true,
        backgroundDecoration: BoxDecoration(
          color: Theme.of(context).canvasColor,
        ),
        loadingChild: Center(
          child: CircularProgressIndicator(),
        ),
      ),
    );
```
And when I rotate and move my image it goes wild. It looks like the anchor point of that image is placed somewhere at the bottom right of my screen. Not the image. It works anyway but it's very uncomfortable to work with image like this.
My screen: 1280x800, my image: 600x300 
Galaxy Tab S4

What's wrong with my attempt of applying this plugin?

**Comment by gppam** [comment]
I think it's the enableRotation that's causing the problem. [Here's my situation too.](https://imgur.com/a/hvAmUt7)

**Comment by ProgrammingLife** [comment]
@gppam I see exactly the same behaviour that you are showing at the video.

**Comment by renancaraujo** [comment]
Released a fix on 0.6.0
